compose/hubmap.yml

```
@@ -7,9 +7,11 @@ services:
     container_name: portal-ui
     image: hubmap/portal-ui:latest
 
-    # Mount the app config in container to keep it outside the image.
     volumes:
+      # Mount the app config in container to keep it outside the container.
       - "../context/instance/app.conf:/app/instance/app.conf"
+      # Mount the logs so they are visible outside the container.
+      - "/var/log/portal-ui/:/var/log/"
```

```suggestion
      - "/home/centos/hubmap/portal-ui/log/:/var/log/"
```
@yuanzhou 
> if you just mount (using the relative path) the nginx logging (access log and error log) to a sub-directory under portal-ui project root directory, that should work well across all deployment environments.

Is this what you'd prefer? I think this is really deployment configuration, and you can take the lead.